dataactvalidator/validation_handlers/validator.py

```diff
@@ -750,14 +752,7 @@ def validateTAS(fieldsToCheck, tasFields, record, interfaces, fileType):
 
         for i in range(0,len(fieldsToCheck)):
             data = record[str(fieldsToCheck[i])]
-            if(data == None):
-                # Set data to empty string so it can be padded with leading zeros
-                data = ""
-            field = fieldsToCheck[i].lower()
-            # Pad field with leading zeros
-            length = interfaces.validationDb.getColumnLength(field, fileType)
-            data = data.strip().zfill(length)
-            query = query.filter(TASLookup.__dict__[tasFields[i]] == str(data))
+            query = query.filter(TASLookup.__dict__[tasFields[i]] == data)
```

Comment:
That padding is now done in the preprocessing before the data is put into staging.  When I added it there I did not think to remove it here, but this is now duplicate functionality.